[2022-12-11 16:52:30,085] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T16:52:27.198227+00:00 [queued]>
[2022-12-11 16:52:30,104] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_dag.world manual__2022-12-11T16:52:27.198227+00:00 [queued]>
[2022-12-11 16:52:30,105] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 16:52:30,107] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-12-11 16:52:30,108] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-12-11 16:52:30,135] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): world> on 2022-12-11 16:52:27.198227+00:00
[2022-12-11 16:52:30,143] {standard_task_runner.py:52} INFO - Started process 3479 to run task
[2022-12-11 16:52:30,156] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_dag', 'world', 'manual__2022-12-11T16:52:27.198227+00:00', '--job-id', '1041', '--raw', '--subdir', 'DAGS_FOLDER/first-dag.py', '--cfg-path', '/tmp/tmpvxn339it', '--error-file', '/tmp/tmpkw2gkvar']
[2022-12-11 16:52:30,160] {standard_task_runner.py:80} INFO - Job 1041: Subtask world
[2022-12-11 16:52:30,279] {task_command.py:369} INFO - Running <TaskInstance: first_dag.world manual__2022-12-11T16:52:27.198227+00:00 [running]> on host 7e39590a7807
[2022-12-11 16:52:30,415] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_dag
AIRFLOW_CTX_TASK_ID=world
AIRFLOW_CTX_EXECUTION_DATE=2022-12-11T16:52:27.198227+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-11T16:52:27.198227+00:00
[2022-12-11 16:52:30,417] {logging_mixin.py:115} INFO - Airflow
[2022-12-11 16:52:30,418] {python.py:173} INFO - Done. Returned value was: None
[2022-12-11 16:52:30,443] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_dag, task_id=world, execution_date=20221211T165227, start_date=20221211T165230, end_date=20221211T165230
[2022-12-11 16:52:30,485] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-12-11 16:52:30,547] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
